tests/documentation/test_files/DocumentationStyleBear/bad_file.py

```diff
@@ -0,0 +1,11 @@
+def hello_there(x):
+    """
+    hello
+    :param x:
+         5 space indent
+        just a second test
+    :return:
+       3 space indent
+        following lines are 4 space indented
+    """
+    return None
```

Comment:
though this is at least a good test, we should actually also assert for the diff/corrected version, otherwise we don't know what actually happens :D

but this can be also done after merge :+1: